mabwiser/_version.py

```diff
@@ -3,5 +3,5 @@
 
 __author__ = "FMR LLC"
 __email__ = "opensource@fmr.com"
-__version__ = "2.5.0"
+__version__ = "2.5.1"
```

Comment:
since this is a new capability, looks like a 2.6 to me

Reply:
In my mind this is an extension of the previous PR and fundamentally covers the same warm_start() capability?

Reply:
Exactly my reasoning

Reply:
kinda disagree (and it could be because I am not close..) but if I am reading the PR right, it introduces a compeletely new field that was not there before; mabwiser.cold_arms OR as we brainstorm in the other thread mabwiser.arm_to_status.. 

My rationale is (IF) this is net new, then it's not an amendment/fix to something existing, hence not 2.5.1 and 2.6.0, but it does not break anything backward or change the paradigm going forward hence not 3.0.0 and 2.6.0

Reply:
your call, OK either way

Reply:
With the arm to status changes, I've bumped to 2.6.0